migration_guide_provisioning.md

```diff
@@ -0,0 +1,176 @@
+# Azure IoT Device SDK for Python Migration Guide - ProvisioningDeviceClient -> ProvisioningSession
+
+This guide details how to update existing code for IoT Hub provisioning that uses an `azure-iot-device` V2 release to use a V3 release instead.
+
+**Note that currently V3 only presents an async set of APIs. This guide will be updated when that changes**
+
+**Note that ProvisioningSession is still not yet available as it is in development**
+
+For changes when communicating between a device and IoT Hub, please refer to `migration_guide_iothub.md` in this same directory.
+
+## Provisioning using Shared Access Key (Symmetric Key)
+Using shared access key authentication (formerly called symmetric key) is now provided via the `shared_access_key` parameter on the `ProvisioningSession` constructor. rather than having it's own `create_from_symmetric_key()` factory method.
```

Comment:
This information is wrong based on the last discussion we had. I had the class method create from symmetric key but now I don't have it anymore.